Cargo.toml

```diff
@@ -10,6 +10,157 @@ members = [
     "libs/*",
 ]
 
+[workspace.package]
+edition = "2021"
+license = "Apache-2.0"
+
+## All dependency versions, used in the project, grouped in some arbitrary order.
```

Comment:
Can we not have them grouped in some arbitrary order? My personal order of preference:

alphabetical >> chaos >> obscure partitions

Reply:
Sorted.
Had left a few small groups below, such as `## TODO replace this with tracing` one.